lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -522,7 +527,7 @@ HdDirtyBits HdVP2Mesh::GetInitialDirtyBitsMask() const
     constexpr HdDirtyBits bits = HdChangeTracker::InitRepr | HdChangeTracker::DirtyPoints
         | HdChangeTracker::DirtyTopology | HdChangeTracker::DirtyTransform
         | HdChangeTracker::DirtyMaterialId | HdChangeTracker::DirtyPrimvar
-        | HdChangeTracker::DirtyVisibility | HdChangeTracker::DirtyInstanceIndex
+        | HdChangeTracker::DirtyVisibility | HdChangeTracker::DirtyInstancer
```

Comment:
Does it need to be protected with the macro conditional compilation?

Reply:
Nope, ok here too, similar to https://github.com/Autodesk/maya-usd/pull/889#discussion_r518217643.